### DISCUSSION TITLE
Allow for editing of logins/credentials through the CLI

### DISCUSSION THREAD

**Discussion by bruno-brant** [Dashlane/dashlane-cli#269] [Ideas]
My list of logins was build through many providers over the years (3 browsers, other vaults, etc.) and now contains many duplicates. The web interface, no matter how great, will never suffice as a tool for mass editing. If the CLI had actions to read & update login data (email, url, etc.) one could easily put a script together that would find duplicates and merge them.

**Comment by Mikescops** [comment]
Hello, thanks for the suggestion, it's quite a piece of work to allow editing but it's on top of our mind for future steps yes.
